src/modules/highlight_follow_streamer_chat/index.js

```diff
@@ -0,0 +1,77 @@
+const settings = require('../../settings');
+const watcher = require('../../watcher');
+const twitch = require('../../utils/twitch');
+const twitchAPI = require('../../utils/twitch-api');
+
+const STYLE_HIGHLIGHT_FOLLOW = 'bttv-highlight-follow-nick';
+const STYLE_HIGHLIGHT_STREAMER = 'bttv-highlight-streamer-nick';
+
+const followsArray = [];
+let currentChannelName;
+let secondRequestAllowed = true;
+
+function putFollowsInArray(direction, limit) {
```

Comment:
i would not feel comfortable with the amount of requests this generates.. every page load could cause many records being loaded from twitch, and this doesn't even cache that data

Reply:
@night I can add a cache. But will this be enough? There will still be many requests.
You need to decide. If the cache helps with this problem, then I'll do it. But if it doesn’t make sense, then I can just close the pull request.